app/models/chargeback/consumption_history.rb

```diff
@@ -42,13 +38,7 @@ def self.base_rollup_scope
         :parent_storage     => :tags,
         :parent_ems         => :tags)
                                 .select(*Metric::BASE_COLS).order('resource_id, timestamp')
-
-      perf_cols = MetricRollup.attribute_names
-      rate_cols = ChargebackRate.where(:default => true).flat_map do |rate|
-        rate.chargeback_rate_details.map(&:metric).select { |metric| perf_cols.include?(metric.to_s) }
-      end
-      rate_cols.map! { |x| VIRTUAL_COL_USES[x] || x }.flatten!
-      base_rollup.select(*rate_cols)
+      base_rollup.select(*ChargeableField.chargeable_cols_on_metric_rollup)
```

Comment:
benefit probably is not so big, just one place for selecting columns and little more readable `base_rollup` 
but it is ok.

hmm as I am looking on `base_rollup_scope`  it can be scope method on `MetricRollup`